base_wamas_ubl/lib/wamas/wamas_grammar/art.py

```diff
@@ -315,7 +315,7 @@
             "type": "bool",
             "length": 1,
             "dp": False,
-            "dict_key": "sell_only_by_packaging",
+            "dict_key": "compulsory_lot",
```

Comment:
Art_ChargePflWa must be "N"
Art_ChargePflWe must be             "dict_key": "compulsory_lot",

Reply:
I updated